apps/extension/src/background/approvals/service.ts

```diff
@@ -321,7 +318,7 @@ export class ApprovalsService {
     if (allowConnection) {
       await addApprovedOrigin(this.approvedOriginsStore, interfaceOrigin);
```

Comment:
Not sure if `await this.keyRingService.connect(interfaceTabId);` cant throw an error anymore, but maybe it's worth to move this `await addApprovedOrigin(this.approvedOriginsStore, interfaceOrigin);` below it. Otherwise  we can add origin even if  connection potentially fails? I did this in the airdrop branch

Reply:
Fixed! Thanks!